components/About.tsx

```diff
@@ -1,15 +1,37 @@
-import React from "react";
+import React, { useEffect, useRef } from "react";
+
 import styled from "@emotion/styled";
 import ReactMarkdown from "react-markdown";
 
 type Props = {
   description: string;
 };
+
 export const About: React.FC<Props> = ({ description, ...delegated }) => {
+  const plugins = useRef<any[]>([]);
+  useEffect(() => {
+    async function loadPlugins() {
+      const remarkBreaks = (await import("remark-breaks")).default;
+      plugins.current = [remarkBreaks];
+    }
+    loadPlugins();
+  }, []);
+
   return (
     <Wrapper {...delegated}>
       <ReactMarkdown
-        components={{ h1: Heading, p: Paragraph, ul: List, ol: List }}
+        remarkPlugins={plugins.current}
+        components={{
+          h1: Heading,
+          h2: Subheading,
+          h3: Subheading,
+          h4: Subheading,
+          h5: Subheading,
+          h6: Subheading,
+          p: Paragraph,
+          ul: List,
+          ol: List,
+        }}
```

Comment:
would be great if we could see some screens for this to compare against the before pic in clubhouse issue